### DISCUSSION TITLE
About cocos studio. Is it deprecated?

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#1160] [Q&A] [answered]
@halx99
As far as I know you be the developer of cocos studio. Is it deprecated?

**Comment by halx99** [comment, marked as answer]
I'm not developer of cocostudio, but should be deprecated, and as extension of axmol can be simply disable by users

**Comment by aismann** [comment]
Opps.
🎉x-studio 2022 is it, not cocos studio. My mistake.

**Reply by halx99** [reply]
No, the `x-stduio 2023` is comming and will migrate to x64 edition. by the way, I think maybe it's time to mark axmol windows x86_32 build deprecated.
